Non-Hermetic Magic Items.md

```diff
@@ -71,3 +70,10 @@ Corpse Magic 35
 This jar contains hundreds of restless spirits that, when the jar is opened, seek a suitable corpse to animate. The corpses are under the control of the user for Sun duration and, when dusk/dawn arrives, all the spirits return into the jar leaving the corpses.  
 The jar contains 7 pawns of Corpus vis and it was found by a temple priest of the Cult of Dedun while searching the initiation script to become the Undying Ones, in a pyramid.  
 *(Base 15, +5 Sun, +10 Group, +5 Size)*
+
+**Detached Head of Wisdom**  
```

Comment:
Where is the object used ? Planned for Rojyar at a later age?

Reply:
It was for a Learned Magician Cultist of Dedun that I created.

Reply:
Which you haven't added to the repo, isn't it ?

Reply:
> 

No, I haven't made him yet.